client/post-editor/post-editor.jsx

```diff
@@ -343,11 +323,26 @@ const PostEditor = React.createClass( {
 								site={ site }
 								type={ this.props.type }
 							/>
+							<div className="editor__site">
```

Comment:
This class would be a bit misleading given that it also includes the status. I'd look for a better name.